### DISCUSSION TITLE
A complaint

### DISCUSSION THREAD

**Discussion by Erogunaiye** [community/community#18160] [GitHub Education] [answered]
I've submitted a response concerning the application for the GitHub Education benefits... I submitted my admission letter but I am yet to receive a positive mail saying it has been activated

**Comment by d33kshant** [comment, marked as answer]
once your student pack is actiated you will recive a welcome email on you registred student email id. In some cases it can take sevral days to activate your github student pack after uploading a valid student id proof. In my case it took around 20hrs.
